client/app/models/ListingModel.js

```diff
@@ -37,14 +37,15 @@ const parseListingImages = (images) => new ListingImage({
   square2x: images.square2x,
 });
 
-export const parse = (l) => {
+export const parse = (l, formListingPath) => {
```

Comment:
the naming is a bit obscure, maybe `listingPathRoute` or `getListingPath` or something?